_posts/2017-02-02-Rust-1.15.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: post
+title: "Announcing Rust 1.15"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.15.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.15.0][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.15.0][notes] on GitHub.
+X patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1150-2017-02-02
+
+### What's in 1.15.0 stable
+
+Rust 1.15 sees an _extremely_ long-awaited feature land on stable: custom
+derive! To review, in Rust, you could automatically implement a trait through
+the `derive` attribute:
+
+```rust
+#[derive(Debug)]
+struct Pet {
+    name: String,
+}
+```
+
+The `Debug` trait would be implemented for `Pet`. However, this only worked for
+traits provided as part of the standard library; it was not customizable. With
+Rust 1.15, it now is. That means, if you want to turn your `Pet` into JSON, it's
+as easy as adding [serde] to your `Cargo.toml`:
+
+```toml
+[dependencies]
+serde = "0.9.5"
```

Comment:
It's not linked in the code example below though? (something needs to change, I'm not sure what)